srv/srv0start.cc

```
@@ -397,21 +380,21 @@ ib_err_t InnoDB::start() noexcept {
   temporarily clear m_config.m_file_per_table.  This is ok,
   because the server will not accept connections (which could
   modify m_config.m_file_per_table) until this function has returned. */
-  srv_config.m_file_per_table = false;
+  state.srv_config.m_file_per_table = false;
 
   IF_DEBUG(log_warn("!!!!!!!! UNIV_DEBUG switched on !!!!!!!!!");)
```

cppcheck

[error] There is an unknown macro here somewhere. Configuration is required. If ; is a macro then please configure it.

(unknownMacro)